animations/lib/main.dart

```diff
@@ -11,6 +11,7 @@ import 'src/basics/04_tweens.dart';
 import 'src/basics/05_animated_builder.dart';
 import 'src/basics/06_custom_tween.dart';
 import 'src/basics/07_tween_sequence.dart';
+import 'package:animations/src/basics/08_hero_animation.dart';
```

Comment:
This app is using path-based imports, so new imports should stick to that pattern.